### DISCUSSION TITLE
How to change + choose VAE

### DISCUSSION THREAD

**Discussion by 49Wares** [lllyasviel/Fooocus#1253] [Q&A]
Hello folks, 
sorry for the stupid question, but how can i change the vae in Fooocus.
I see the folder vae and insert vae here file, but after i put my vae there, how can i select which one to use?
I could not find any information on that.
Best regards

**Comment by xhoxye** [comment]
cannot change

**Reply by 49Wares** [reply]
Hello, thanks for the reply.
But what is the folder good for than?
And why is there a file inside that says something like: "put vae here".
Best regards

**Reply by xhoxye** [reply]
At least up to now, it is not a choice. Maybe it is to prepare for the future.
